### ISSUE TITLE
Battlegrounds battles not updating to homepage

### ISSUE THREAD

**Issue by saphyron** [HearthSim/Hearthstone-Deck-Tracker#4457]
**Describe the bug**
My latest 2 games has not been uploaded to the homepage.

**To Reproduce**
1. Go to 'homepage my stats'
2. Compare latest games to session Recap
3. See error

**Game mode**
- [x] Battlegrounds

**Expected behavior**
Games updating

**Screenshots**
![image](https://user-images.githubusercontent.com/30288325/226758935-286985b4-d491-47ac-9132-f1577404392f.png)
![image](https://user-images.githubusercontent.com/30288325/226758978-352b198e-3b71-4357-b8d2-9d1cda4c7596.png)

**Version (please complete the following information):**
I think it is Version 1.19.16 Not sure if its the correct one, but its what I found in help section

**Log file**
No such folder on my pc.

**Comment by JackRammer** [comment]
same on the overview page "My Games". After the match the result is not shown immediately as some time ago

**Comment by Ploute** [comment]
Yep problem persists with version 1.20.10 too. Some games it's not showing the available minion types, or the results after the game.
